.sync.yml

```diff
@@ -1,64 +1,15 @@
 ---
 Gemfile:
-  required:
-    ':system_tests':
-      - gem: 'puppet-module-posix-system-r#{minor_version}'
-        platforms: ruby
-      - gem: 'puppet-module-win-system-r#{minor_version}'
-        platforms:
-          - mswin
-          - mingw
-          - x64_mingw
-      - gem: beaker-testmode_switcher
-        version: '~> 0.4'
-      - gem: master_manipulator
-      - gem: puppet-blacksmith
-        version: '~> 3.4'
   optional:
     ':development':
       - gem: 'github_changelog_generator'
         git: 'https://github.com/skywinder/github-changelog-generator'
         ref: '20ee04ba1234e9e83eb2ffb5056e23d641c7a018'
         condition: "Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.2.2')"
 
-
-.gitignore:
-  required:
-    - ---.project
-
-spec/default_facts.yml:
-  unmanaged: true
-
-# The registry module doesn't support parallel_spec
-appveyor.yml:
```

Comment:
does appveyor need ignored ? will pdk update smash the file ?

Reply:
I was expecting `pdk update` to do the right thing - I'll need to investigate appveyor/litmus integration more.

Reply:
copied over the recommended files.